contrib/opencensus-ext-azure/opencensus/ext/azure/common/utils.py

```diff
@@ -23,19 +23,35 @@
 from opencensus.common.version import __version__ as opencensus_version
 from opencensus.ext.azure.common.version import __version__ as ext_version
 
-azure_monitor_context = {
-    'ai.cloud.role': os.path.basename(sys.argv[0]) or 'Python Application',
-    'ai.cloud.roleInstance': platform.node(),
-    'ai.device.id': platform.node(),
-    'ai.device.locale': locale.getdefaultlocale()[0],
-    'ai.device.osVersion': platform.version(),
-    'ai.device.type': 'Other',
-    'ai.internal.sdkVersion': 'py{}:oc{}:ext{}'.format(
-        platform.python_version(),
-        opencensus_version,
-        ext_version,
-    ),
-}
+_azure_monitor_context = {}
+
+
+def get_azure_monitor_context(
+        role_name=(
```

Comment:
How come these are default kwargs? Shouldn't these just be assignments in the function?

Reply:
@lzchen , that was part of the original code (default values). Since the azure_monitor_context is used in potentially many different places, I decided to keep the current default values. Makes sense?

Reply:
Yes keeping the default values makes sense. I meant since `get_azure_monitor_context()` is a function now, having kwargs implies that you can actually instantiate these values as different values (instead of just the constants), which actually shouldn't be allowed. If we are always calling it with no parameters, why even have the kwargs? Just assign them to default values in the actual function.

Reply:
oh, yes. sure. the reason for providing default arguments, is because of testability. Having method arguments makes it easy to validate that the monitor context will contain the correct values (and we don't have to deal with mocking env variables, etc). What do you think.

But method args also provides some additional flexibility (if needed) .

Reply:
No we don't really want to have changed functionality for this. Users are not meant to change the values of the context.